### PR TITLE
Issue 4 journal about bible verses. - Justin Son

### PR DESCRIPTION
**Pull request by justintaeson**
https://user-images.githubusercontent.com/97268025/157534916-e26a2f4c-6920-4f51-bdd9-3f05b59e3f74.mov

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -156,28 +169,78 @@ function saveVerse(event) {
...
+$journalForm.addEventListener('submit', renderEntry);
+
+function renderEntry(event) {
+  event.preventDefault();
+  var $textArea = document.querySelector('textarea');
+  var $innerEntryDiv = document.getElementById('saved-journals');
+  $innerEntryDiv.textContent = $textArea.value;
+  $innerEntryDiv.className = 'margin-bottom padding-sides';
+
+  data.entries[data.nextEntryId - 1].entry = $textArea.value;
```
Comment (codingcodymiller) [inline js/main.js]: Woah, why not just use `push`?

File: js/main.js [hunk 2]
```diff
@@ -156,28 +169,78 @@ function saveVerse(event) {
...
+      var $p = document.createElement('p');
+      $p.id = 'journal-verse';
+
+      $ul.appendChild($li);
+      $li.appendChild($p);
+      $li.id = data.nextEntryId - 1;
+      $p.textContent = data.entries[data.nextEntryId - 1].verse;
+    } else {
+      var $pVerse = document.getElementById('journal-verse');
+      $pVerse.textContent = data.entries[data.nextEntryId - 1].verse;
```
Comment (codingcodymiller) [inline js/main.js]: Using `data.nextEntryId` to determine the length of the array is a bad idea, if you ever add the ability to delete entries, then the next id will not match the actual count of items stored.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: js/main.js (hunk 1 above)
Comment (justintaeson) [reply js/main.js]: Hey Cody, I'm not trying to push anything here. I'm just trying to access an index of the data.entries array. The goal is try and find a previous journal entry and populate into the text area when a user wants to revisit their journal entry and edit it again.

**Comment by justintaeson** [comment]
https://user-images.githubusercontent.com/97268025/157564404-aab9dd0a-d0e9-4cf5-987e-6bea22ec74ce.mov

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**